test/unit/Dataflow_utils.ml

```diff
@@ -119,7 +123,11 @@ let%expect_test "Loop passthrough" =
         }
       |}
   in
-  let mir = Ast_to_Mir.trans_prog "" (semantic_check_program ast) in
+  let typed_ast, warnings = semantic_check_program ast in
```

Comment:
Would you mind making a helper function to pull this out? Something very close to this stanza is repeated a lot in these test files.